com.unity.render-pipelines.universal/Documentation~/complex-lit-shader.md

```diff
@@ -0,0 +1,90 @@
+# Complex Lit Shader
+
+The Complex Lit Shader is similar to the Lit shader, but it adds advanced material features. These features often are considerably more resource intensive and therefore are only supported on relatively new hardware. Complex Lit shader is always rendered using forward rendering and used as a fallback shader in deferred rendering. Complex Lit contains all the features that are incompatible with deferred rendering. This Shader uses the most computationally heavy [shading model](shading-model.md) in the Universal Render Pipeline (URP). 
+
+## Using the Complex Lit Shader in the Editor 
+
+To select and use this Shader:
+
+1. In your Project, create or find the Material you want to use the Shader on.  Select the __Material__. A Material Inspector window opens. 
+2. Click __Shader__, and select __Universal Render Pipeline__ > __Complex Lit__.
+
+## UI overview  
+
+The Inspector window for this Shader contains these elements: 
+
+- __[Surface Options](#surface-options)__
+- __[Surface Inputs](#surface-inputs)__
+- __[Advanced](#advanced)__
+
+![Inspector for the Lit Shader](Images/Inspectors/Shaders/Lit.png)
+
+
+
+### Surface Options 
+
+The __Surface Options__ control how URP renders the Material on a screen. 
```

Comment:
I wonder if there's a way to "inherit" docs from Lit shader and just document what's added/new in this one. Otherwise we might desync these docs fairly quickly (Lukas is just having another PR open that adds detail and parallax to lit f.ex). @oleks-k do you have ideas?

Reply:
Yes, I'll see if I can change this with [!include] files.